frontend/src/app/modules/aspect-configuration/services/aspect-configuration.service.ts

```diff
@@ -109,6 +116,9 @@ export class AspectConfigurationService {
   private getPerfAspectParams(): Observable<any> {
     return combineLatest(this.applicationService.selectedApplication$, this.selectedPage$)
       .pipe(
+        distinctUntilChanged((previousSelection: [Application, Page], currentSelection: [Application, Page]) => {
```

Comment:
`distinctUntilChanged` with a `compare` function makes sense when we compare `previousSelection` with `currentSelection` here. But this is not the case.
If we do not need to compare those two variables, wouldn't a `filter` function sufice?